content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -169,6 +170,7 @@
 public class ResourcesAction 
 	extends PagedResourceHelperAction // VelocityPortletPaneledAction
 {
+	 private EventTrackingService eventTrackingService;
```

Comment:
This has to be declared using the API.

private static final EventTrackingService eventTrackingService = ComponentManager.get(EventTrackingService.class);